gui/default/index.html

```diff
@@ -910,7 +910,7 @@ <h4 class="panel-title">
                       </tr>
                       <tr ng-if="deviceCfg.introducedBy">
                         <th><span class="far fa-fw fa-handshake-o"></span>&nbsp;<span translate>Introduced By</span></th>
-                        <td class="text-right">{{ deviceName(devices[deviceCfg.introducedBy]) || deviceCfg.introducedBy.substring(0, 5) }}</td>
+                        <td class="text-right">{{ deviceName(devices[deviceCfg.introducedBy]) || deviceCfg.introducedBy.substring(0, shortIDStringLength) }}</td>
```

Comment:
The function `deviceName` already returns a short-id if there is no name, so the or part here does nothing and can be removed. Then you don't need to add `shortIDStringLength` to `$scope`.

Reply:
@imsodin Should I really remove it? There seems to be handling for `undefined` with `""` which is then handled by that `||` and that one isn't present in `deviceName()` :thinking:

Reply:
Oh no, you are right - I misread that code. I was thinking the check was done on the same config, but it's `devices[deviceCfg.introducedBy]` vs plain `deviceCfg` afterwards. So lets leave the `||` part, however you can use the existing `deviceShortID` function instead without needing `shortIDStringLength` here.

Reply:
@imsodin Fixed :v: